portability-gateway/src/main/java/org/dataportabilityproject/gateway/reference/ReferenceApiServer.java

```diff
@@ -66,6 +81,37 @@ public void stop() {
     server.stop(0);
   }
 
+  private HttpServer createServer(int port) throws Exception {
```

Comment:
Can HttpServer be injected, and the local extension somehow provides this version whereas default provides the old version?